### PR TITLE
Feature send reports sqs

### PR DESCRIPTION
**Pull request by kjgarza**
This is ready for review

### CODE REVIEW

File: app/controllers/concerns/helpeable.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+module Helpeable
+  extend ActiveSupport::Concern
+
+  require 'active_support/core_ext/object/try'
+  require 'action_controller'
+
+  included do
+
+    def self.permit_recursive_params(params)
+      Rails.logger.info "ccjhchch"
```
Comment (mfenner) [inline app/controllers/concerns/helpeable.rb]: What is this?

File: app/controllers/concerns/helpeable.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      # Rails.logger.info params.inspect
+      (params.try(:to_unsafe_h) || params).map do |key, value|
```
Comment (mfenner) [inline app/controllers/concerns/helpeable.rb]: What are you trying to do here? It looks overly complicated.

File: app/models/report.rb [hunk 3]
```diff
@@ -24,6 +27,14 @@ class Report < ApplicationRecord
   serialize :exceptions, Array
   before_create :set_id
   before_validation :set_uid, on: :create
+  after_create :pust_report
```
Comment (mfenner) [inline app/models/report.rb]: push?

File: app/controllers/concerns/helpeable.rb (hunk 1 above)
Comment (kjgarza) [reply app/controllers/concerns/helpeable.rb]: sorted in 0623cd0

File: app/controllers/concerns/helpeable.rb (hunk 2 above)
Comment (kjgarza) [reply app/controllers/concerns/helpeable.rb]: this has been changed as well  in 0623cd0

File: app/models/report.rb (hunk 3 above)
Comment (kjgarza) [reply app/models/report.rb]: typo fixed in 423f0f6

**Comment by kjgarza** [comment]
yes; I have changed this 42046340bcf2da52a

**Pull request merged**